INSTALL

```diff
@@ -157,10 +157,13 @@
                    "--cross-compile-prefix=mipsel-linux-gnu-" suffices
                    in such case. Needless to mention that you have to
                    invoke ./Configure, not ./config, and pass your target
-                   name explicitly.
+                   name explicitly. Even keep in mind that --openssldir
```

Comment:
Remove the "Even".

Reply:
I kind of want to make connection to "Needless to mention". Would "and" do instead of "even"?